rp/frontend/src/lib/components/ViewExclusiveContentModal.svelte

```diff
@@ -40,7 +39,6 @@
         groupName: credentialName,
         owner,
         identity: $authStore.identity,
-        toastStore,
```

Comment:
This parameter is not expected by the function. I wonder why TS didn't complain before.